src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -7473,6 +7710,8 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+      isMemberOfGroup:
+        type: boolean
```

Comment:
What is this `isMemberOfGroup` boolean property used for? Are you planning to actually resolve all groups and return all members of all groups as individual site members?

Reply:
it is just Boolean property that helps to identify the user is part any group or not